From fd937b064aec587fa2860e8ad15c3a573fb2a366 Mon Sep 17 00:00:00 2001
From: Andreas Rehn <rehn.andreas86@gmail.com>
Date: Wed, 17 Jul 2024 21:49:52 +0200
Subject: [PATCH] u-boot: add support for Lichee Pi Zero Dock

Upstream-Status: Inappropriate [custom machine not upstreamed]
---
 arch/arm/dts/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 9d28a485be..2a886f5cd9 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -808,7 +808,8 @@ dtb-$(CONFIG_MACH_SUN8I_V3S) += \
 	sun8i-s3-elimo-initium.dtb \
 	sun8i-s3-pinecube.dtb \
 	sun8i-v3-sl631-imx179.dtb \
-	sun8i-v3s-licheepi-zero.dtb
+	sun8i-v3s-licheepi-zero.dtb \
+	sun8i-v3s-licheepi-zero-dock.dtb
 dtb-$(CONFIG_MACH_SUN8I_R528) += \
 	sun8i-t113s-mangopi-mq-r-t113.dtb
 dtb-$(CONFIG_MACH_SUN50I_H5) += \
